src/components/fileSelect/FileSelect.tsx

```diff
@@ -10,7 +10,6 @@
 
 import React, {ChangeEvent} from "react";
 
-import "styles/components/FileSelect.scss";
```

Comment:
Please don't remove styles from components

Reply:
Since we use `tsc` for bundling, it won't generate the _styles_ folder in the build.  If we are importing the styles in the component, we will get an error stating that "the styles folder is not found", while consuming the component in the product. So, instead of importing styles in the component, we are making use of the `main.css`, which would have the styles for all the components in the library.